opensearch/src/test/java/org/opensearch/sql/opensearch/request/OpenSearchRequestBuilderTest.java

```diff
@@ -72,18 +82,13 @@ public class OpenSearchRequestBuilderTest {
 
   @BeforeEach
   void setup() {
-    when(settings.getSettingValue(Settings.Key.QUERY_SIZE_LIMIT)).thenReturn(200);
-    when(settings.getSettingValue(Settings.Key.SQL_CURSOR_KEEP_ALIVE))
-        .thenReturn(TimeValue.timeValueMinutes(1));
-
-    requestBuilder = new OpenSearchRequestBuilder(
-        "test", MAX_RESULT_WINDOW, settings, exprValueFactory);
+    requestBuilder = new OpenSearchRequestBuilder(DEFAULT_LIMIT, exprValueFactory);
   }
 
   @Test
   void build_query_request() {
-    Integer limit = 200;
-    Integer offset = 0;
+    int limit = 200;
```

Comment:
why?

Reply:
`int` is more appropriate in this case but didn't realize it makes it inconsistent with the rest of the file. I'll change it back for that reason.

A common use case for `Integer` class is store integer values in `List` classes and similar but for simple use cases like `limit` and `offset` in this case it just adds overhead.